### PR TITLE
Add basic tooltips

### PR DESCRIPTION
**Pull request by jacobbearden**
Appearance:
tooltip appearance

Markup:
```html
<span class="tooltip">
  hover me
  <span class="tooltip-body">tooltip</span>
</span>
```

### CODE REVIEW

File: bulma/components/tooltip.sass [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.tooltip
+  position: relative
+  cursor: help
+  &:hover .tooltip-body
+    opacity: 1
+    right: -20px
```
Comment (jgthms) [inline bulma/components/tooltip.sass]: What if there's no space on the right side?

File: bulma/components/tooltip.sass [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.tooltip
+  position: relative
```
Comment (jgthms) [inline bulma/components/tooltip.sass]: What if the element is already positioned (through another class) or has `overflow: hidden`?

File: bulma/components/tooltip.sass [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  cursor: help
+  &:hover .tooltip-body
+    opacity: 1
+    right: -20px
+  .tooltip-body
+    background: #222324
+    border-radius: 3px
+    color: white
+    font-size: 12px
+    margin-right: -200px // width
```
Comment (jgthms) [inline bulma/components/tooltip.sass]: What if goes out of the viewport?

File: bulma/components/tooltip.sass [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.tooltip
+  position: relative
+  cursor: help
+  &:hover .tooltip-body
+    opacity: 1
+    right: -20px
+  .tooltip-body
+    background: #222324
```
Comment (jgthms) [inline bulma/components/tooltip.sass]: Use a variable.

File: bulma/components/tooltip.sass [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    position: absolute
+    right: 9999px
+    top: -4px // top padding
+    transition: opacity 250ms ease-out
+    width: 200px
+    word-wrap: break-word
+    &:before
+      border: 6px solid transparent
+      border-right-color: #222324
+      content: ' ' // must have content to display
```
Comment (jgthms) [inline bulma/components/tooltip.sass]: Double quotes please.

**Comment by jgthms** [comment]
You forgot to use the autoprefixer and removed all vendor prefixes.

**Comment by mesqueeb** [comment]
Dear Bulma developers,
do the tooltips like in bootstrap work in Bulma?

**Pull request closed without merging**